maraboupy/test/test_nnet_extensions.py

```diff
@@ -0,0 +1,149 @@
+
+from maraboupy.MarabouNetworkNNet import *
+
+
+from subprocess import call
+
+property_filename = "./resources/properties/acas_property_4.txt"
+network_filename = "./resources/nnet/acasxu/ACASXU_experimental_v2a_1_9.nnet"
+
+layer = 2
+
+
+def test_write_to_file():
+    output_filename = "./maraboupy/test/ACASXU_experimental_v2a_1_9_output.nnet"
+
+    nnet_object = MarabouNetworkNNet(filename=network_filename)
```

Comment:
It's better practice to use Marabou.read_nnet( filename ) instead of calling MarabouNetworkNNet directly. Users only need to import maraboupy.Marabou to do anything they want with Maraboupy, whether it's reading an NNet file or tensorflow file, and these tests should be a good model for how Maraboupy should be used.

Reply:
Done. 

BTW, is there a reason why some of the imports were relative? (e.g., from .Marabou import.. )? I have changed all of them to maraboupy.Marabou etc, otherwise things don't work on my system (but I can try to fix this issue, if the relative import is important).

Reply:
Sometimes the relative imports are helpful if the Marabou repo has not been added to PYTHONPATH, and the user is not in the Marabou repo folder. What errors were you seeing with relative imports? You're right that we are not consistent about doing it one way or another, which is something we should address in the future.

Reply:
They just were not working for me for some reason (python did not find the modules). I think that it would be good to agree on a uniform policy, and if we decide to stick with relative imports, I will see how to fix the issue on my Mac/PyCharm combination.

Speaking of uniformity: there are several (but very few) methods that are named using a different (perhaps a more "pythonian") naming convention, for historical reason, I assume: read_nnet, for instance. Should we consider changing this, maybe? Or will this cause too much trouble in too many scripts of too many people in too many different repositories at this point?